src/sidebar/index.js

```diff
@@ -52,9 +52,9 @@ function setupApi(api, streamer) {
  */
 // @inject
 function setupRoute(groups, session, router) {
-  Promise.all([groups.load(), session.load()]).finally(() => {
-    router.sync();
-  });
+  groups.load();
+  session.load();
+  router.sync();
 }
```

Comment:
The `setupRoute` function is a misnomer. I think it would make sense to move all service initialization into one function in future to reducer boilerplate and make it easier to see the order in which the various init steps happen. I didn't do that in this PR to avoid conflating a functional change with a refactoring.

Reply:
Yes, it pops out at me, too. Let's rename when it's convenient.